### DISCUSSION TITLE
Postal and Outlook coexistence - How to DNS Setup?

### DISCUSSION THREAD

**Discussion by Pacerino** [postalserver/postal#2678] [Help with using Postal] [answered]
Hello together!

First of all, thank you for this great software!
We're running Microsoft 365 in our Company to send and recieve mails. For our system and applications I want to implement postal but i'm not sure how to setup the DNS and Postal correctly so Outlook and Microsoft can coexist.

What we have:
Domain example.org -> pointing to Micrsoft
Domain mail.example.org -> pointing to Postal
All DNS requirements from the Postal docs have been setup. Eg. rp.mail.example.org, spf.mail.example.org etc.

If i want to **send only** i can setup the example.org domain and send from it. Inbound mails are getting routed to Outlook because of the MX record to Outlook.
if i want to **send and recieve** mails in postal, the sender domain has to be mail.example.org and a MX record to mail.example.org

Is that correct?

Thanks!

**Comment by willpower232** [comment, marked as answer]
That is correct yes.

There will be a return path for all emails sent from Postal so still one MX record involved but that would be psrp.example.org I think.

**Reply by Pacerino** [reply]
i appreciate your help!
